src/app/query/hiro-config/hiro-config.query.ts

```diff
@@ -54,3 +61,17 @@ export function useHasFiatProviders() {
     Object.keys(activeProviders).reduce((acc, key) => activeProviders[key].enabled || acc, false)
   );
 }
+
+export function useConfigFeeEstimationsEnabled() {
+  const config = useRemoteHiroConfig();
+  if (typeof config?.feeEstimations === 'undefined') return;
```

Comment:
In `app/utils` we have `isNumber` & `isString` methods. Maybe you can add one for undefined, too.

Reply:
Sure !